spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/blocking/retry/BlockingLoadBalancedRetryFactory.java

```diff
@@ -33,13 +34,31 @@
 
 	private final LoadBalancerProperties loadBalancerProperties;
 
+	private LoadBalancerPropertiesFactory factoryProperties;
+
+	@Deprecated
```

Comment:
Please add javadoc with `@deprecated` tag indicating the suggested replacement.

Reply:
Added